gui/autobutcher.lua

```diff
@@ -709,6 +734,29 @@ function WatchList:onButcherRace()
     self:refresh()
 end
 
+function WatchList:onUnbutcherAll()
+    for _, data in ipairs(plugin.autobutcher_getWatchList()) do
+        plugin.autobutcher_unbutcherRace(data.id)
+    end
+
+    self:refresh()
+end
+
+function WatchList:onButcherAll()
+    dlg.showYesNoPrompt(
```

Comment:
After playing around with the UI changes, I'm going back and forth on whether this prompt is necessary. Marking all animals for butchering is non-destructive -- you can easily unmark all for butchering.

However, if a player *accidentally* hits `e`, then it could be an unpleasant surprise when all the player's animals are subsequently slaughtered.

Butchering all animals should be an infrequent enough action such that the prompt probably won't become annoying, so I think it can stay.